charts/testkube-cloud-api/templates/deployment.yaml

```diff
@@ -101,15 +101,15 @@ spec:
             - name: SMTP_PORT
               value: "{{ required ".Values.api.smtp.port is required with .Values.api.inviteMode set to \"email\"" .Values.api.smtp.port }}"
```

Comment:
Is there any reason why the port is required and does not use the 25 which is SMTP default?

Reply:
For the required properties above, the relationship is also completely reversed. `inviteMode` requires SMTP, SMTP does not require `inviteMode`; there are other reasons to send emails.